### ISSUE TITLE
Crash on startup when installing add-ons importing gui at installTasks module level

### ISSUE THREAD

**Issue by nvdaes** [nvaccess/nvda#11950]
### Steps to reproduce:

1. Install an add-on which imports gui in installTasks at the module level, such as [reportPasswords 1.0](https://github.com/nvdaes/reportPasswords/releases/download/1.0/reportPasswords-1.0.nvda-addon)
 
### Actual behavior:
NVDA crashes and a message is presented informing that NVDA needs to be running to install add-ons.

This maybe produced by a circular import related to gui.contextHelp, according to this [old issue](https://github.com/nvaccess/nvda/issues/4496). It seems to be fixed following the suggestion provided there by @jcsteh , that is, importing gui at the onInstall() method level. 

### Expected behavior:
NVDA should be restarted without crashing.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2020.4beta1
#### Windows version:
10
#### Name and version of other software in use when reproducing the issue:
n/a
#### Other information about your system:
n/a
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. In previous versions NVDA is restarted without crashing.
#### If addons are disabled, is your problem still occuring?
Yes.
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes.

**Comment by nvdaes** [comment]
[nvda-old.log](https://github.com/nvaccess/nvda/files/5719073/nvda-old.log)

**Comment by lukaszgo1** [comment]
Hi @nvdaes 
I've created #11951 to fix this. Would you be able to give a build from this PR a try and confirm if this issue is fixed for you?

**Comment by nvdaes** [comment]
Replying to @lukaszgo1 
> Would you be able to give a build from this PR a try and confirm if this issue is fixed for you?

Unfortunately not at this moment, since I don't have installed required software to build NVDA. If I can, I will create an action to create a build using GitHub Actions from your PR later or tomorrow.
Thanks so much.

**Comment by nvdaes** [comment]
Hi @lukaszgo1, your PR fixes this issue according to my test with an NVDA built from source using GitHub Actions.
Test performed:

- Install NVDA built from #11951. 
- - Install reportPassword 1.0.
Results:

NVDA is restarted normally.

Thanks.

**Comment by lukaszgo1** [comment]
@nvdaes Thanks for confirming. For reference  every pull request opened against NVDA repository is compiled on AppVeyor and by clicking 'details' after the build status you can download a compiled launcher from the artifact section - there is no need to build it with gitHub actions or anything so complicated.

**Comment by nvdaes** [comment]
Thank you.

**Comment by feerrenrut** [comment]
Closed with 6e161eb (PR #11958)
